apostrophecms/a3-docs
# adds APOS_BASE_URL to docs

## Summary

*Summarize the changes briefly, including which issue/ticket this resolves. If it closes an existing Github issue, include "Closes #[issue number]"*
This PR updates the a3 docs to include the `APOS_BASE_URL` environment variable. Closes PRO-3776 for the a3 docs

## What are the specific steps to test this change?

*For example:*
> 1. Run the website and log in as an admin
> 2. Open a piece manager modal and select several pieces
> 3. Click the "Archive" button on the top left of the manager and confirm that it should proceed
> 4. Check that all pieces have been archived properly
1. Build the docs
2. Navigate to `/reference/modules/i18n.html#locales`, `/reference/modules/login.html#passwordreset`, and `/reference/modules/page.html#getbaseurl-req`

## What kind of change does this PR introduce?
*(Check at least one)*


## Make sure the PR fulfills these requirements:


If adding a new feature without an already open issue, it's best to open a **feature request issue** first and wait for approval before working on it.

**Other information:**
